pkg/imgpkg/imageset/unprocessed_image_refs.go

```diff
@@ -11,37 +11,50 @@ import (
 	regname "github.com/google/go-containerregistry/pkg/name"
 )
 
+const RootBundleLabelKey string = "dev.carvel.imgpkg.copy.root-bundle"
+
 type UnprocessedImageRef struct {
 	DigestRef string
 	Tag       string
+
+	Labels map[string]string
+}
+
+func (u UnprocessedImageRef) Key() string {
+	return u.DigestRef + u.Tag
```

Comment:
just in case, lets add a separator / between the two since we are combining two unrelated pieces into a single data.

Reply:
ok. done!